src/DynamoPackages/Package.cs

```diff
@@ -527,6 +532,27 @@ internal void MarkForUninstall(IPreferences prefs)
             RaisePropertyChanged(nameof(LoadState));
         }
 
+        /// <summary>
+        /// Marks any given package for unload.
+        /// The package will not be marked for deletion.
+        /// </summary>
+        /// <param name="prefs"></param>
+        internal void MarkForUnload(IPreferences prefs)
```

Comment:
As far as I remember, we cannot really schedule non-built in packages to be unloaded (unload without deleting from disk)
Only built in packages can be unloaded, non-built in packages will be uninstalled (deleted from disk)
The function that has this logic is `MarkForUninstall` - handles both built-ins and non-built-ins